source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -5,4 +5,140 @@ Access Data From a Cursor
 .. default-domain:: mongodb
 
 Read operations that return multiple documents do not immediately return
-all values matching the query.
+all values matching the query. Because a query can potentially match
+large sets of documents, these operations rely upon a
+:java-docs:`FindIterable
+<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`.
+
+A ``FindIterable`` iterates through all the documents in your collection
+to find documents that match your search criteria. It also allows you to
+further specify which documents to see by setting paramaters though methods. 
+
+This page uses an initiating method, ``find()`` to show how to access
+data from a ``FindIterable``. 
+
+Terminal Methods
+----------------
+
+Execute database operations on a ``FindIterable`` using terminal
+methods. Terminal methods return a single value after iterating thorugh
```

Comment:
Not sure what is meant by `return a single value`.  I guess MongoCursor is a single value, but it's confusing to think of it that way since you have to further iterate through it.  Perhaps something like:

"Terminal methods execute the operation on the MongoDB server after all parameters controlling the operation have been configured on the `Iterable` instance.